packages/react-scripts/config/webpack.config.js

```diff
@@ -284,7 +284,7 @@ module.exports = function(webpackEnv) {
       alias: {
         // Support React Native Web
         // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
-        'react-native': 'react-native-web',
+        '^react-native$': 'react-native-web',
```

Comment:
Is the leading caret necessary? I don't think aliases are actual regexes, and I don't see it documented [here](https://webpack.js.org/configuration/resolve/), only the trailing `$`.

Reply:
Good catch, I removed it.